model/actions/GetActiveDeliveryExecution.php

```diff
@@ -69,8 +70,9 @@ public function __invoke($params)
             if (empty($executions)) {
                 $active = $this->getTool()->startDelivery($this->delivery, $remoteLink, $user);
             } else {
+                $resumable = $this->getServiceLocator()->get(DeliveryServerService::SERVICE_ID)->getResumableStates();
                 foreach ($executions as $deliveryExecution) {
-                    if (!$deliveryExecutionService->isFinished($deliveryExecution)) {
+                    if (in_array($deliveryExecution->getState()->getUri(), $resumable)) {
```

Comment:
I think it's better to update `\oat\ltiDeliveryProvider\model\execution\implementation\AbstractLtiDeliveryExecutionService::isFinished` to use ` getResumableStates()` inside of it because `isFinished` method was created exactly for this purpose (see phpdoc: https://github.com/oat-sa/extension-tao-ltideliveryprovider/blob/master/model/execution/LtiDeliveryExecutionService.php#L38)